### PR TITLE
added page object model project

### PR DESCRIPTION
**Pull request by TurgayArif**
Added project with page object model

### CODE REVIEW

**Review by balantaim — COMMENTED** [review]
error: The following untracked working tree files would be overwritten by checkout:
	target/test-classes/TestSuites/AllTests.xml
Please move or remove them before you switch branches.

1. I think you should unstage target/test-classes/TestSuites/AllTests.xml
I cannot switch to your branch
2. This is very important please rename your branch as example idnumber_nameofthetask_ourname -> example 99_pageobjectmodel_turgay

**Comment by TurgayArif** [comment]
> 

1. You need to force your checkout with "git checkout -f turgay"
This happened because I added the "target" folder in the .gitignore file and your local file with name "AllTests.xml" is tracked by git.
2. We don't have task with ID for the page object model, so I will leave it as it is for now.

Let me know if the force checkout worked for you.

**Comment by desislavavelichkova** [comment]
I can run only test "HomePageTests".

**Comment by desislavavelichkova** [comment]
Please change "FullUITestSuite" with "AllTests" in TestSuites.AllTests.xml file. Everything works properly with that changing.

File: src/test/resources/TestSuites/AllTests.xml [hunk 1]
```diff
@@ -1,11 +1,43 @@
...
-<suite name="AllTests" verbose="1">
-	
-	<test name="FirstTest">
-		<classes>
-			<class name="com.easyrestaurant.tests.FirstTest" />
-		</classes>
-	</test>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="FullUITestSuite" verbose="1">
```
Comment (desislavavelichkova) [inline src/test/resources/TestSuites/AllTests.xml]: Change "FullUITestSuite" with "AllTests" and all test will be runned.

**Review by dashov — APPROVED** [review]

**Review by desislavavelichkova — APPROVED** [review]

**Review by balantaim — APPROVED** [review]
If start the tests with suite file everithing is OK.

**Pull request merged**